test/common/http/filter/fault_filter_test.cc

```diff
@@ -413,4 +413,46 @@ TEST_F(FaultFilterTest, FixedDelayAndAbortHeaderMatchFail) {
   EXPECT_EQ(0UL, config_->stats().aborts_injected_.value());
 }
 
+TEST_F(FaultFilterTest, TimerResetAfterStreamReset) {
+  SetUpTest(fixed_delay_only_json);
+
+  // Prep up with a 5s delay
+  EXPECT_CALL(runtime_.snapshot_, featureEnabled("fault.http.delay.fixed_delay_percent", 100))
+      .Times(1)
```

Comment:
nit: Times(1) is implied when you do .WillOnce() or just a plain expect_call, so can typically be removed. There are multiple cases in here where this applies.